### ISSUE TITLE
connecting two esp8266

### ISSUE THREAD

**Issue by timdicus** [esp8266/esp8266-wiki#45]
Hi there, can we set one esp as server other as client using arduino ide and then make two arduinos communicate via esp8266?

**Comment by camilstaps** [comment]
Yes. One ESP8266 could act as an access point, the other one as a client. If there is a wireless network you can connect to, you can use both as clients.

However, if you just need to connect two Arduinos, the ESP8266 might be an overkill. It includes the whole TCP/IP stack, which is some overhead, consumes more power than a simple 433MHz transmitter, etc. On the other hand, having a TCP/IP stack means automatic packet loss handling and more fancy things.
